packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/index.md

```diff
@@ -27,26 +27,43 @@ Add authentication using the Okta [redirect model](https://developer.okta.com/do
 
 ## Set up Okta
 
-Set up your [Okta org](/docs/concepts/okta-organizations/). The CLI is by far the quickest way to work with your Okta org, so we recommend using it for the first few steps. If you don't want to install the CLI, you can [manually sign up for an org](https://developer.okta.com/signup/) instead. We provide non-CLI instructions along with the CLI steps below.
+Set up your [Okta org](/docs/concepts/okta-organizations/). The Okta command-line interface (CLI) is the quickest way to work with your Okta org, so we recommend using it for the first few steps. If you don't want to install the CLI, you can [manually sign up for an org](https://developer.okta.com/signup/) instead.
 
-1. Install the Okta command-line interface: [Okta CLI](https://cli.okta.com/).
-2. If you don't already have a free Okta developer account, create one by entering `okta register` on the command line.
-3. Make a note of the Okta Domain as you need that later.
-4. **IMPORTANT:** Set the password for your Okta developer org by opening the link that's shown after your domain is registered. Look for output similar to this:
+1. Install [Okta CLI](https://cli.okta.com/).
+1. If you don't already have a free Okta developer account:
+   1. Open your terminal.
+   {style="list-style-type:lower-alpha"}
+   1. Run `okta register`, and enter your first name, last name, email address, and country.
+   1. Click or tap **Activate** in the account activation email that is sent to the email address that you gave.
 
-   ```
-   Your Okta Domain: https://dev-xxxxxxx.okta.com
-   To set your password open this link:
-   https://dev-xxxxxxx.okta.com/welcome/xrqyNKPCZcvxL1ouKUoh
-   ```
+      > **Tip**: If you don't receive the confirmation email sent as part of the creation process, check your spam filters for an email from `noreply@okta.com`
 
-   > **Note**: If you don't receive the confirmation email sent as part of the creation process, check your spam filters for an email from `noreply@okta.com`.
+   1. After your domain is registered, look for output similar to this:
```

Comment:
Maybe something like:

Details about your Okta domain are output after your domain is registered:

Reply:
Changed to    1. Find your new domain and a link to set your password in the email: